clients/py/README.md

```diff
@@ -20,6 +20,8 @@ w3.middleware_onion.add(construct_sign_and_send_raw_middleware(account))
 
 # Finally, wrap the provider to add Sapphire end-to-end encryption
 w3 = sapphire.wrap(w3)
+# Optionally, select the gas price oracle implemented in the Oasis Web3 Gateway
+w3.eth.set_gas_price_strategy(rpc_gas_price_strategy)
```

Comment:
Where is `rpc_gas_price_strategy` defined? Should this be `sapphire.rpc_gas_price_strategy`?

Reply:
This is from [`web3`](https://github.com/ethereum/web3.py/blob/8f853f5841fd62187bce0c9f17be75627104ca43/web3/gas_strategies/rpc.py#L14). I'll add the import.